playbooks/operationalizing/cloudforms.adoc

```diff
@@ -223,6 +223,35 @@ Under *Endpoints*, select the *Hawkular* tab and enter the hawkular-metrics host
 
 Click *Add* to add the container provider.
 
+==== CloudForms 4.2:
+
+In the CloudForms web console, hover over *Compute* in the menu on the left, then *Containers*, then select *Providers*.
+
+Select the *Configuration* button and then *Add a New Containers Provider*.
+
+Enter a name for the provider and select *OpenShift Enterprise* in the type (ignore the zone, this will populate on its own). 
+
+Under *Endpoints*, in the *Default* tab, enter the hostname or address of the HAProxy server (or the master in a single master setup) into the *Hostname* field and enter the token retrieved earlier into the *Token* and *Confirm Token* fields. Click *Validate* to confirm connectivity.
```

Comment:
My assumption is that this is for the Master API and not the HAProxy router. If so, we should probably rephrase to mention the more generic load balancer in front of the masters.

Since it is my assumption that you used the 4.1 instructions as a base, if you make changes here, would you be able to replicate for the 4.1 instructions